packages/expo-cli/e2e/__tests__/init-test.ts

```diff
@@ -0,0 +1,30 @@
+import path from 'path';
+
+import JsonFile from '@expo/json-file';
+import temporary from 'tempy';
+
+import { runAsync, tryRunAsync } from '../TestUtils';
+
```

Comment:
might want to set up beforeEach and afterEach blocks to create a temporary directory and set process.env.PWD to the temp dir, and delete the temp dir when done so that the test cleans up after itself

Reply:
I considered this, when writing the test, but decided against cleaning up the temporary directory for a few reasons:
- the test sets `$PWD` to a temporary folder inside `/tmp` so the operating system will automatically clean it up afterwards, on reboot
- cleaning up would therefore add unnecessary complexity to the test suite
- if there's a failure, it's a lot easier to debug, if the folder is still there after the test finishes.